kolibri/plugins/coach/assets/src/vue/groups-page/delete-group-modal.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+
+  <core-modal :title="$tr('deleteLearnerGroup')"
+    @cancel="close">
+    <p>{{ $tr('areYouSure') }} <strong>{{ groupName}}</strong>?</p>
```

Comment:
@rtibbles I remember running into some issues with `$trHtml` before, but I think I was just using the `reverseString` flag we had set in options. Did that "translation" not behave the same way as another language would? Would people translating need to be careful to include the HTML tags? If so, can we expect them to?